### PR TITLE
feat(proposer): rm tier fee

### PR DESCRIPTION
**Pull request by YoGhurt111**
ref: 
https://github.com/taikoxyz/taiko-mono/pull/17740
https://github.com/taikoxyz/taiko-mono/pull/17750
https://github.com/taikoxyz/taiko-mono/pull/17748

### CODE REVIEW

**Comment by YoGhurt111** [comment]
Wait for taiko-client version updated

**Comment by davidtaikocha** [comment]
> Wait for taiko-client version updated

`0.29.0` tag is online, can change the tag in docker-compose files now.

File: .env.sample [hunk 1]
```diff
@@ -67,10 +65,6 @@ ENABLE_PROPOSER=false
 L1_PROPOSER_PRIVATE_KEY=
 # Address of the proposed block's suggested L2 fee recipient.
 L2_SUGGESTED_FEE_RECIPIENT=
-# Fee sent to prover for the off-chain bond (in Gwei).
-BLOCK_PROPOSAL_FEE=1
-# If nobody accepts block at initial tier fee, how many iterations to increase tier fee before giving up.
-MAX_TIER_FEE_BUMPS=
 # Comma-delineated list (no spaces) of prover endpoints proposer should query when attempting to propose a block
 # If you keep this default value you must also enable a prover by setting ENABLE_PROVER=true.
 # If you start only proposer - add provers endpoint from provers maklet site: https://dojonode.xyz/tatami (wait for load and click "copy All" and paste instead of http://taiko_client_prover_relayer:9876)
```
Comment (RogerLamTd) [inline .env.sample:70]: rm this and line below?

Comment (YoGhurt111) [reply .env.sample:70]: Yes!

**Review by davidtaikocha — APPROVED** [review]

**Pull request merged**